src/textual/dom.py

```diff
@@ -192,6 +190,14 @@ def __init_subclass__(
         cls._merged_bindings = cls._merge_bindings()
         cls._css_type_names = frozenset(css_type_names)
 
+    async def _pre_process(self) -> None:
```

Comment:
`Reactable` initialization / teardown has moved from `MessagePump` to `DOMNode`. `MessagePump` cannot be a reactable because it does not have `refresh`, so better be explicit about these.